saleor/site/models.py

```diff
@@ -159,3 +164,56 @@ def get_translated_keys(self):
             "header_text": self.header_text,
             "description": self.description,
         }
+
+
+class SiteByIdLoader(DataLoader):
```

Comment:
Wouldn't be better to migrate loaders to `saleor/site/dataloaders.py`? This will keep consistency with the way how we store current data loaders.
Helper methods could be placed in a separate file or (IMHO) it could be also placed in the file with dataloaders.

Reply:
Yep, data loaders are GraphQL specific, so we should put them somewhere under `saleor/graphql`, model files are not the correct layer.